modules/ROOT/pages/errors/gql-errors/42NAA.adoc

```diff
@@ -0,0 +1,44 @@
+= 42NAA
+
+== Status description
+error: syntax error or access rule violation - incorrectly formatted graph reference. Incorrectly formatted graph reference `{ <<input>> }`. Expected a single quoted or unquoted identifier. Separate name parts should not be quoted individually.
+
+== Explanation
+As of Cypher 25, database names and aliases cannot include name parts that are quoted individually.
+Either the whole name must be quoted, or the whole name must be unquoted.
+
+== Example scenario
+For example, try to create a database with :
+
+[source,cypher]
+----
+CREATE DATABASE `foo`.`bar`
+----
+
+An error will be thrown with GQLSTATUS 42NAA and the status description:
+
+[source]
+----
+error: syntax error or access rule violation - incorrectly formatted graph reference. Incorrectly formatted graph reference '`foo`.`bar`'. Expected a single quoted or unquoted identifier. Separate name parts should not be quoted individually.
+----
+
+In this example scenario, the correct quoting would either be
+
+[source,cypher]
+----
+CREATE DATABASE `foo.bar`
+----
+
+or
+
+[source,cypher]
+----
+CREATE DATABASE foo.bar
+----
+
```

Comment:
Perhaps we want to mention parameter and/or link to the graph reference docs here as well